feat: override uuid during init (#1963)
defenseunicorns/pepr: docs/030_user-guide/120_customization.md

| Field            | Description                            | Example Values                  |
|------------------|----------------------------------------|---------------------------------|

> | `uuid`           | Unique identifier for the module*      | `hub-operator`                  |

Reviewer: ```suggestion
| `uuid`           | Unique identifier for the module       | `hub-operator`                  |
```

Does that `*` add any specific meaning?

Author: It is supposed to draw attention to the longer explanation below on 138. May not not be necessary though. We can scrap it. I am noticing we may need an extra space though when we scrap it.

| `rbac`           | Custom RBAC rules (requires building with `rbacMode: scoped`)            | `{"rbac": [{"apiGroups": ["<apiGroups>"], "resources": ["<resources>"], "verbs": ["<verbs>"]}]}` |
| `rbacMode`       | Configures module to build binding RBAC with principal of least privilege | `scoped`, `admin` |

> **uuid**: This is used to identify the module in the `pepr-system` namespace. If not provided, a UUID will be generated. It can be any [kubernetes acceptable name](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/) that is under 36 characters.

Reviewer: ```suggestion
**uuid**: An identifier for the module in the `pepr-system` namespace. If not provided, a UUID will be generated. It can be any [kubernetes acceptable name](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/) that is under 36 characters.
```
